test/media_addon_workflow_spec.rb

```diff
@@ -0,0 +1,49 @@
+#! /usr/bin/env rspec
+
+require_relative "spec_helper"
+require "yaml"
+
+describe "Registration::UI::MediaAddonWorkflow" do
```

Comment:
I see the same problem here already mentioned in other tests. It actually looks to me just like a 1:1 reproduction of the implementation. But in this case I'm not so certain about how to fix it.